client/app/components/composites/Menu/Menu.css

```diff
@@ -0,0 +1,85 @@
+.menu {
+  font-family: var(--topbarFontFamily); /* should come from topbar */
+  font-size: var(--Menu_fontSize);
+  flex-grow: 1;
+
+  &:focus,
+  & :focus {
+    outline: none;
+  }
+
+  &:hover .menuContent {
+    outline: none;
+    display: block;
+  }
+}
+
+.menuLabel {
+  display: inline-block;
+  font-size: var(--Menu_fontSize);
+  padding: 28px 24px;
+  white-space: nowrap;
+  cursor: pointer;
+
+  &:first-letter {
+    text-transform:capitalize;
+  }
+}
+
+.menuLabelIcon {
+  margin-right: var(--Menu_iconPadding);
+}
+.menuLabelDropdownIcon {
+  margin-left: 0.2857em;
+}
+
+.menuLabelDropdownIconOpen {
+  composes: menuLabelDropdownIcon;
+  display: none;
+}
+.menu:hover .menuLabelDropdownIconOpen {
+  display: inline-block;
+}
+
+.menuLabelDropdownIconClosed {
+  composes: menuLabelDropdownIcon;
+  display: inline-block;
+}
+.menu:hover .menuLabelDropdownIconClosed {
+  display: none;
+}
+
+.menuContent {
+  display: none;
+  z-index: 1;
+  background-color: var(--Menu_colorBackground);
+  border: 1px solid var(--Menu_borderColor);
+  border-radius: 3px;
+  position: absolute;
+  box-shadow: var(--Menu_boxShadow);
+}
+
+.menuContentArrowTop,
+.menuContentArrowBelow {
+  bottom: 100%;
+  left: 50%;
+  border: solid transparent;
+  content: " ";
+  height: 0;
+  width: 0;
+  position: absolute;
+  pointer-events: none;
```

Comment:
"pointer-events allows authors to control under what circumstances (if any) a particular graphic element can become the target of mouse events"
(but this is autogenerated stuff from http://www.cssarrowplease.com )